### PR TITLE
Added branch name to slide out dialog

### LINKED ISSUE #736: Branch name is absent from slide-out dialog
...which may lead to someone misclicking and removing the wrong branch.

### PR DESCRIPTION
**Pull request by DcortezMeleth**
Resolves #736

### CODE REVIEW

**Review by PawelLipski — APPROVED** [review]
Tested locally... works fine!

File: frontendResourceBundles/src/main/resources/GitMacheteBundle.properties [hunk 1]
```diff
@@ -131,8 +131,12 @@ action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.title=Git
...
-action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-1=Git Machete is able to delete the local branches after sliding them out.
-action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-2=\
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-1=\
+<html> \
+  You are going to slide out branch <b>{0}</b>.\
+</html>
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-2=Git Machete is able to also delete the local branches after sliding them out.
+action.GitMachete.BaseSlideOutBranchAction.deletion-suggestion-dialog.note-3=\
 <html> \
   You can change it by setting the git config <b>{0}</b> to <b>true</b> or <b>false</b>. \
```
Comment (PawelLipski) [inline frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:141]: Hmmm actually probably a pre-existing problem with the trailing text being cropped:

image

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:141]: (`false` missing 🤔)

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:141]: And responding to:

> Also one more thought. Would it be beneficial to add some information on this dialog, that Remember my choice is not some Intellij default behaviour, but rather our own addition, which under the hood adds git config entry like this: machete.slideOut.deleteLocalBranch = true?

Hmm it is kind of suggested by _You can change it..._, but I agree that's not 100% unambiguous 🤔

Comment (DcortezMeleth) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:141]: Sorry for slow response. I will check today why this part is missing. Which resolution did you use?

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:141]: 1920x1080

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:141]: np! 🇳🇵

Comment (PawelLipski) [reply frontendResourceBundles/src/main/resources/GitMacheteBundle.properties:141]: FYI, changed to sth like this:

image

**Pull request merged**